src/test/java/gov/usds/case_issues/config/OAuthMappingConfigTest.java

```diff
@@ -153,11 +171,28 @@ public void createDelegatingUserService_nameInList_correctUser() {
 	public void createDelegatingUserService_nameInScalar_correctUser() {
 		OAuth2UserService<OAuth2UserRequest, OAuth2User> service = setupService("my_attr", "scalar_name");
 		OAuth2User user = service.loadUser(null);
-		assertEquals("name_in_scalar;04c6752c-31cd-437a-b7a3-70f25676af1b", user.getName());
+		assertEquals("name_in_scalar", user.getName());
 		assertEquals(simpleAttributes(), user.getAttributes());
 		assertEquals(Collections.singleton(new SimpleGrantedAuthority("respect")), user.getAuthorities());
 	}
 
+	@Test
+	public void createDelegatingUserService_newUser_userCreated() {
```

Comment:
It's possible that we can/should make this use a mocked `UserService`, and rely on the UserService tests to validate the correctness of the behavior once the service itself is called, but that seems only modestly worth the trouble at the moment.